### PR TITLE
Feature/create-social-api

### PR DESCRIPTION
**Pull request by jc-hiroto**

### CODE REVIEW

File: src/routes/social.js [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+  }else if(post.downvotes.includes(user_id)){
+    return -1;
+  }else{
+    return 0;
+  }
+}
+
+
+router.get('/posts/:id/', checkJwt, async (req, res) => {
+  // get course social posts by course id
```
Comment (swh00tw) [inline src/routes/social.js:24]: by post id, not by course id?

Comment (jc-hiroto) [reply src/routes/social.js:24]: This method is created for future-proof. There is a GET post by course_id method below! 👌

Comment (jc-hiroto) [reply src/routes/social.js:24]: Ahh... I see.
You're talking about the comment right? I'll fix it...🥵

Comment (jc-hiroto) [reply src/routes/social.js]: Modified comment.

**Review by swh00tw — COMMENTED** [review]
GJ!!

**Pull request merged**